.env

```
@@ -0,0 +1,18 @@
+﻿# Variables de entorno para el proyecto LTI
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The ﻿# Variables de entorno para el proyecto LTI key has incorrect delimiter

(IncorrectDelimiter)